src/components/modules/forms/Login/index.tsx

```diff
@@ -0,0 +1,16 @@
+import { Formik, Form, Field } from "formik";
+import { useLogin } from "hooks/useLogin";
+
+export const LoginForm = () => {
+    const { onSubmit, initialLoginValues } = useLogin();
+
+    return (
+        <Formik initialValues={initialLoginValues} onSubmit={onSubmit}>
+            <Form>
+                <Field type={'email'} name={'email'} placeholder={'Email'} />
```

Comment:
<Field type="email" name="email" placeholder="Email" />
If you're passing string values do not use {}

Reply:
What is more, I suggest, create "translations" directory with en.json file storing all available text values. 
Then use it with i18n. If you do this in the early step of the project, you wouldn't do a lot of changes when adding additional translations. It's good for future improvments